src/main/scala/com/ambiata/disorder/N.scala

```diff
@@ -11,9 +11,12 @@ import GenPlus._
 case class N(value: String)
 
 object N {
+
+  val newlines: List[Char] = List('\n', '\r', 0x000B, 0x000C, 0x0085, 0x2028, 0x2029)
+
   implicit def NArbitrary: Arbitrary[N] =
     Arbitrary(for {
       a <- arbitrary[S]
-      c <- checkGen[Char](arbitrary[Char], "Char", x => x != '\n' && x != '\r')
-    } yield N(a.value.replace('\n', c).replace('\r', c)))
+      c <- checkGen[Char](arbitrary[Char], "Char", x => !newlines.contains(x))
+    } yield N(newlines.foldLeft(a.value)({ case (b, a) => b.replace(a, c) })))
```

Comment:
There isn't a way of replacing them all in one pass is there by any happy chance? /nit